code-blast.js

```diff
@@ -204,7 +204,7 @@ https://twitter.com/JoelBesada/status/670343885655293952
 			cmNode = cm.getWrapperElement();
 			init();
 		} else {
-			destroy();
+			if(cm) destroy();
```

Comment:
How can such a case arise? Also `destroy` shouldn't have a check on `cm`. Instead the `cm.off` statement should be checked for `cm` inside the `destroy` function so that rest of the destroy proceeds as usual.